tools/js-tools/git-hooks/pre-commit-hook.js

```diff
@@ -119,10 +119,10 @@ function sortPackageJson( jsFiles ) {
 }
 
 const gitFiles = parseGitDiffToPathArray(
-	'git -c core.quotepath=off diff --cached --name-only --diff-filter=ACM'
+	'git -c core.quotepath=off diff --cached --name-only --diff-filter=ACMR'
 ).filter( Boolean );
 const dirtyFiles = parseGitDiffToPathArray(
-	'git -c core.quotepath=off diff --name-only --diff-filter=ACM'
```

Comment:
For anyone who like me may be wondering, `R` stands for renamed. :) Seems to make sense here.